app/models/ability.rb

```diff
@@ -86,10 +86,8 @@ def signed_in(user)
       event.users.include?(user)
     end
 
-    # cannot create an event if program does not have open cfp
-    cannot [:new, :create], Event do |event|
-      user_inclusion = event.event_users.map { |event_user| event_user.user.id }.compact.include? user.id
-      !event.program.cfp_open? || !event.new_record? || !user_inclusion
+    can [:new, :create], Event do |event|
+      event.program.cfp_open? && event.new_record?
     end
```

Comment:
I know, I've read the readme ;)

Anyway, creating an event with someone else as submitter was possible before, too! And checking user.include doesn't work for new and create!